kie-remote/src/main/java/org/kie/remote/command/SnapshotOnDemandCommand.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.remote.command;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+public class SnapshotOnDemandCommand extends AbstractCommand implements VisitableCommand, Serializable {
```

Comment:
Please add this class to classes forbidden for replicas in https://github.com/kiegroup/openshift-drools-hacep/blob/master/kie-remote/src/test/java/org/kie/remote/command/RemoteCommandsTest.java#L59

Reply:
ok